### PR TITLE
Remove deprecated IceBox.ServiceManager OA

### PR DESCRIPTION
**Pull request by bernardnormier**

### CODE REVIEW

File: cpp/src/IceGrid/NodeCache.cpp [hunk 1]
```diff
@@ -830,31 +830,6 @@ NodeEntry::getInternalServerDescriptor(const ServerInfo& info) const
         }
 
         props.push_back(createProperty("IceBox.LoadOrder", servicesStr));
-
-        if (iceVersion != 0 && iceVersion < 30300)
```
Comment (bernardnormier) [inline cpp/src/IceGrid/NodeCache.cpp:834]: This code was for Ice 3.2 (and earlier) servers.

File: cpp/src/IceGrid/NodeCache.cpp [hunk 2]
```diff
@@ -830,31 +830,6 @@ NodeEntry::getInternalServerDescriptor(const ServerInfo& info) const
...
-                props.push_back(createProperty("IceBox.ServiceManager.RegisterProcess", "1"));
-                server->processRegistered = true;
-            }
-            if (!hasProperty(iceBox->propertySet.properties, "IceBox.ServiceManager.Endpoints"))
-            {
-                props.push_back(createProperty("IceBox.ServiceManager.Endpoints", "tcp -h 127.0.0.1"));
-            }
-        }
-        if (!hasProperty(info.descriptor->propertySet.properties, "IceBox.InstanceName") &&
-            hasProperty(iceBox->propertySet.properties, "IceBox.ServiceManager.Endpoints"))
```
Comment (bernardnormier) [inline cpp/src/IceGrid/NodeCache.cpp:854]: This code was for Ice servers < 3.5, since Ice 3.5 has the Process facet.

**Review by externl — APPROVED** [review]
Looks good!

**Review by InsertCreativityHere — APPROVED** [review]
Yep, looks good indeed!

**Comment by bernardnormier** [comment]
Merged with #2659.

**Pull request closed without merging**